src/EasyList/TodoServiceDB.cs

```diff
@@ -47,10 +47,12 @@ public void DisplayTodo(Todo todo)
         public void DisplayAllTodo(TodoOrder todoOrder)
         {
             var _todoList = _todoRepository.GetAllTodo(todoOrder)
-                            .Select(x => new { x.Id, x.Label, x.Description, x.Priority, x.Status, CreateDate = x.CreatedDate.DateTime.ToLocalTime() , DueDate = x.DueDate?.DateTime.ToLocalTime() });
+                            .Select(x => new { x.Id, x.Label, x.Description, x.Priority, x.Status, CreateDate = x.CreatedDate.DateTime.ToLocalTime(), DueDate = x.DueDate?.DateTime.ToLocalTime() });
```

Comment:
Might be better to add the DisplayId  here so we don't need to loop over the items twice

Reply:
If i add DisplayId here ,in console there will be a id column and DisplayId column.so i use that.

Reply:
There should be a way to not display the real id, or did i misunderstand?

Reply:
i update the code will it work?